drone_talk.py

```diff
@@ -0,0 +1,34 @@
+from discord.ext import commands
+from discord.utils import get
+import discord
+import messages
+from roles import HIVE_MXTRESS, DRONE
+from channels import COMMUNICATION_CHANNEL
+import re
+
+def get_user_id(username):
+	drone_id = re.search(r"\d{4}", username).group()
+	return drone_id if drone_id is not None else 0000;
+
+def has_role(member: discord.Member, role: str) -> bool:
+	return get(member.roles, name=role) is not None
+
+class Drone_Talk(commands.Cog):
+
+	def __init__(self,bot):
+		self.bot = bot
+
+	@commands.Cog.listener()
+	async def on_ready(self):
+		print("Drone talk cog online.")
+
+	@commands.Cog.listener()
+	async def on_message(self, message):
+
+		if(message.channel.name == COMMUNICATION_CHANNEL):
```

Comment:
Python ifs do not require brackets around the condition. It does not change semantics, but you generally do not set them.
```suggestion
		if message.channel.name == COMMUNICATION_CHANNEL:
```